src/CSET/operators/collapse.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 Met Office and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Operators to perform various kind of collapse on either 1 or 2 dimensions.
+"""
+
+import iris
+import iris.cube
+import iris.analysis
+
+
+def collapse_1dim(
+    cube: iris.cube.Cube, coordinate: str, method: str, **kwargs
```

Comment:
```suggestion
    cube: iris.cube.Cube, coordinate: str, method: str, additional_percent: float = None, **kwargs
```
So long as we give it a default value, we have have additional named keyword arguments in the function signature.